web/src/scripts/containers/Root/Router.jsx

```diff
@@ -93,6 +93,7 @@ class AppRouter extends Component {
             component={Preferences}
             onEnter={() => {
               _GA('Preferences')
+              window.__isDecoPreferencesWindow = true
```

Comment:
what does this do? does it need to be global?

maybe generalize to windowId or something.